packages/ui/src/components/BareChatInput/index.tsx

```diff
@@ -322,6 +327,22 @@ export default function BareChatInput({
     [handleSelectMention, controlledText]
   );
 
+  // Handle text attachments by inserting them into the input
+  useEffect(() => {
+    const textAttachment = attachments.find(
+      (a): a is TextAttachment => a.type === 'text'
+    );
+    if (textAttachment) {
+      if (controlledText === '') {
+        handleTextChange(`${textAttachment.text}`);
+      } else {
+        handleTextChange(`${textAttachment.text}${controlledText}`);
```

Comment:
is this basically saying, if there is already content in the input, smash that with the quote as a prefix otherwise just add the quote?

Reply:
yep